docs/system-monitoring.md

```diff
@@ -0,0 +1,26 @@
+# System components monitoring
+
+Controller nodes [are isolated](https://docs.k0sproject.io/v1.23.3+k0s.1/architecture/#control-plane) by default, which thus means that a cluster user cannot schedule workloads onto controller nodes.
```

Comment:
We should avoid specific versions in the links. Try this instead:
(architecture.md#control-plane)

Reply:
Done